app/scripts/views/settings/emails.js

```diff
@@ -0,0 +1,156 @@
+      'click .email-refresh.enabled': preventDefaultThen('refresh'),
+      'click .resend': preventDefaultThen('resend')
+    },
+
+    initialize (options) {
+      if (options.emails) {
+        this._emails = options.emails;
+      } else {
+        this._emails = [];
+      }
+    },
+
+    context () {
+      return {
+        buttonClass: this._hasSecondaryEmail() ? 'secondary' : 'primary',
+        emails: this._emails,
+        hasSecondaryEmail: this._hasSecondaryEmail(),
+        hasSecondaryVerifiedEmail: this._hasSecondaryVerifiedEmail(),
+        isPanelOpen: this.isPanelOpen(),
+        newEmail: this.newEmail
+      };
+    },
+
+    beforeRender () {
+      // Only show this view on verified session
+      return this._isSecondaryEmailEnabled();
+    },
+
+    afterRender () {
+      // Panel should remain open if there are any unverified secondary emails
+      if (this._hasSecondaryEmail() && ! this._hasSecondaryVerifiedEmail()) {
+        this.openPanel();
+      }
+    },
+
+    _isSecondaryEmailEnabled () {
+      // Only show secondary email panel if the user is in a verified session and feature is enabled.
+      const account = this.getSignedInAccount();
+      return account.sessionVerificationStatus()
+        .then((res) => {
+          if (! res.sessionVerified) {
+            return this.remove();
+          }
+
+          // If we fail to fetch emails, then this user does not have this feature enabled
+          // and we should not display this panel.
+          return this._fetchEmails()
+            .fail(() => {
+              return this.remove();
+            });
+        });
+    },
+
+    _hasSecondaryEmail () {
+      return this._emails.length > 1;
+    },
+
+    _hasSecondaryVerifiedEmail () {
+      return this._hasSecondaryEmail() ? this._emails[1].verified : false;
+    },
+
+    _onDisconnectEmail (event) {
+      const email = $(event.currentTarget).data('id');
+      const account = this.getSignedInAccount();
+      return account.recoveryEmailDestroy(email)
+        .then(()=> {
+          return this.render()
+            .then(()=> {
+              this.navigate('/settings/emails');
+            });
+        });
+    },
+
+    _fetchEmails () {
+      const account = this.getSignedInAccount();
+      return account.recoveryEmails()
+        .then((emails) => {
+          this._emails = emails.map((email) => {
+            return new Email(email).toJSON();
+          });
+        });
+    },
+
+    refresh: showProgressIndicator(function() {
+      return this.render();
+    }, EMAIL_REFRESH_SELECTOR, EMAIL_REFRESH_DELAYMS),
+
+    resend (event) {
+      const email = $(event.currentTarget).data('id');
+      const account = this.getSignedInAccount();
+      return account.resendEmailCode(email)
+        .then(() => {
+          this.displaySuccess(Strings.interpolate(t('Verification emailed to %(email)s'), { email: email }), {
```

Comment:
@shane-tomlinson what do you mean? the wording?

`Verification emailed to...` 

When sending a signup email, we say:

> A verification link has been sent&nbsp;to %(escapedEmail)s